app/services/LocationService.js

```diff
@@ -383,70 +423,93 @@ export default class LocationServices {
       isBackgroundGeolocationConfigured = true;
 
       if (!status.locationServicesEnabled) {
-        // we need to set delay or otherwise alert may not be shown
-        // setTimeout(
-        //   () =>
-        //     Alert.alert(
-        //       languages.t('label.require_location_services_title'),
-        //       languages.t('label.require_location_services_message'),
-        //       [
-        //         {
-        //           text: languages.t('label.yes'),
-        //           onPress: () => {
-        //             if (isPlatformAndroid()) {
-        //               // showLocationSettings() only works for android
-        //               BackgroundGeolocation.showLocationSettings();
-        //             } else {
-        //               Linking.openURL('App-Prefs:Privacy'); // Deeplinking method for iOS
-        //             }
-        //           },
-        //         },
-        //         {
-        //           text: languages.t('label.no'),
-        //           onPress: () => console.log('No Pressed'),
-        //           style: 'cancel',
-        //         },
-        //       ],
-        //     ),
-        //   1000,
-        // );
+        //we need to set delay or otherwise alert may not be shown
+        setTimeout(
```

Comment:
looks like this is the exact code from above? if so - we can just call the same method here after extracting the block above into a standalone method.